amxmodx/string.cpp

```diff
@@ -989,9 +1001,9 @@ static cell AMX_NATIVE_CALL amx_strtok2(AMX *amx, cell *params)
 	int trim = params[7];
 
 	// ltrim left
-	if (trim & 1 && isspace(string[i]))
+	if (trim & 1 && (spaces = utf8getspaces(string, len)) > 0)
```

Comment:
` + i` missed?

Reply:
`i` starts from 0 here. So, it should be fine.